### ISSUE TITLE
Fails with "invalid options argument"

### ISSUE THREAD

**Issue by cburgmer** [jcoglan/canopy#40]
When running canopy results in an error:

```
canopy mygrammar.peg --lang js
invalid options argument
TypeError: invalid options argument
    at optsArg (MY_DIR/node_modules/canopy/node_modules/mkdirp/lib/opts-arg.js:13:11)
    at mkdirp (MY_DIR/node_modules/canopy/node_modules/mkdirp/index.js:11:10)
    at write (MY_DIR/node_modules/canopy/bin/canopy:32:5)
    at Object.<anonymous> (MY_DIR/node_modules/canopy/bin/canopy:37:3)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:996:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at /usr/local/lib/node_modules/npm/node_modules/libnpx/index.js:268:14
```

```
$ npm --version
6.13.7
$ node --version
v13.11.0
$ head node_modules/canopy/node_modules/mkdirp/package.json
{
  "_args": [
    [
      "mkdirp@1.0.3",
...
```

My assumption is that mkdirp has a breaking change in 1.0.0, and canopy has no version enforced in the package.json.

**Comment by jcoglan** [comment]
This looks like a duplicate of #37 and #38 -- could you check and confirm?

**Comment by cburgmer** [comment]
Yes. I looked through all the issues, but as Github splits PRs from them, I didn't see those :(
Thanks!

**Comment by jcoglan** [comment]
This should now be fixed as part of the v0.4 release; see https://blog.jcoglan.com/2022/04/01/announcing-canopy-0-4/
